src/utils/index.js

```diff
@@ -1,14 +1,28 @@
 var d3 = require('d3');
 
+var findUniqueValues = function(list) {
+  var o = {}, i, l = list.length, r = [];
+  for(i=0; i<l;i+=1) o[list[i]] = list[i];
+  for(i in o) r.push(o[i]);
+  return r;
+};
 
-exports.calculateScales = (chartWidth, chartHeight, xValues, yValues) => {
+exports.calculateScales = function(chartWidth, chartHeight, xValues, yValues)  {
```

Comment:
indeed you do not have to change these 2 lines.

We prefer using `exports.calculateScales = (chartWidth, chartHeight, xValues, yValues) => {`